exercises/space-age/src/example.h

```diff
@@ -0,0 +1,25 @@
+#ifndef EXAMPLE_H
+#define EXAMPLE_H
+
+/* Number of seconds in a Earth year */
+#define SECONDS_ON_EARTH_YEAR 31557600
+
+/* Orbital period of each planet, in Earth years */
+#define MERCURY_YEAR 0.2408467
+#define VENUS_YEAR 0.61519726
+#define MARS_YEAR 1.8808158
+#define JUPITER_YEAR 11.862615
+#define SATURN_YEAR 29.447498
+#define URANUS_YEAR 84.016846
+#define NEPTUNE_YEAR 164.79132
```

Comment:
Any reason why these are in the header file? It seems like these should be private to the implementation and thus in the .c file.